src/wp-includes/class-wp-http-curl.php

```diff
@@ -307,7 +309,8 @@ public function request( $url, $args = array() ) {
 		);
 
 		// Handle redirects.
-		if ( false !== ( $redirect_response = WP_HTTP::handle_redirects( $url, $r, $response ) ) ) {
+		$redirect_response = WP_Http::handle_redirects( $url, $parsed_args, $response );
```

Comment:
We have `$r` and `$parsed_args` conflicts. `$parsed_args` is not defined in this file. 
It seems like the `$r` variable was renamed in https://core.trac.wordpress.org/browser/trunk/src/wp-includes/class-wp-http-curl.php

Reply:
PR pushed to address this